types/underscore/index.d.ts

```diff
@@ -5691,60 +5691,60 @@ declare module _ {
         * Wrapped type `object`.
         * @see _.keys
         **/
-        keys(): _Chain<string>;
+        keys(): _Chain<string, string[]>;
 
         /**
         * Wrapped type `object`.
         * @see _.allKeys
         **/
-        allKeys(): _Chain<string>;
+        allKeys(): _Chain<string, string[]>;
 
         /**
         * Wrapped type `object`.
         * @see _.values
         **/
-        values(): _Chain<any>;
+        values(): _Chain<any, any[]>;
 
         /**
         * Wrapped type `object`.
         * @see _.mapObject
         **/
-        mapObject(fn: _.ListIterator<T, any>): _Chain<T>;
+        mapObject<TResult>(fn: _.ListIterator<T, TResult>): _Chain<TResult, any>;
```

Comment:
Shoulbe be  _Chain<TResult, Dictionary<TResult>>;

Reply:
Done.